pkgs/development/tools/yourkit-java/default.nix

```diff
@@ -0,0 +1,43 @@
+{ fetchzip
+, lib
+, stdenv
+, jdk17
+}:
+let
+  jre = jdk17;
+
+  vPath = v: lib.elemAt (lib.splitString "-" v) 0;
+
+  arch = if stdenv.targetPlatform.system == "x86_64-linux"
+         then "x64"
+         else if stdenv.targetPlatform.system == "aarch64-linux"
+         then "arm64"
+         else throw "Unsupported system";
+in
+stdenv.mkDerivation rec {
+  pname = "YourKit-JavaProfiler";
```

Comment:
Should this match the name of the derivation instead, that is be `yourkit-java` instead?

Reply:
Yes, `yourkit-java` is a name with less confusion.